src/client/pages/Payment/useTrustlyURL.ts

```diff
@@ -0,0 +1,25 @@
+import { useEffect } from 'react'
+import { useLocation } from 'react-router'
+import { useRegisterDirectDebitMutation } from 'data/graphql'
+import { SUCCESS_SUFFIX, FAIL_SUFFIX } from './Payment.constants'
+
+export const useTrustlyURL = () => {
+  const { pathname } = useLocation()
+  const baseURL = `${window.location.origin}${pathname}`
```

Comment:
I wonder if there's any difference of doing this instead 🤔 

```suggestion
  const baseURL = window.location.href
```

I guess `window.location.href` will also include query parameters right?! It feels "weird" getting `baseURL` by using "2 location objects". Ah, whatever. No actions required here; just a random thought.

Reply:
hmm interesting -- we need `window.location` for the origin

then I use react router location since it will make the hook reactive but perhaps it's not really needed...

Reply:
Yeah, we probably want it to be reactive so the correct `baseUrl` gets passed to `useRegisterDirectDebitMutation`. That's fine 😄